plugins/image/config/policy.json

```diff
@@ -1,19 +1,19 @@
 {
   "image_viewer": "role:image_viewer",
-  
-  "image:image_owner": "project_id:%(image.owner)s",
```

Comment:
Are you sure you want to remove the image owner rule? That means that anybody with image_admin can delete any image even ones they haven't created. Also if you do remove the rule you need to remove it from the rules further down as well. Currently it's still being used e.g. in the image_delete rule.